payload-examples/api.github.com/status/1.payload.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
Could you rename this to something more descriptive?
`with-author-committer-null.payload.json`?